### ISSUE TITLE
Can't run WarmStartRoutingSpec

### ISSUE THREAD

**Issue by michaz** [LBNL-UCB-STI/beam#1782]
I can't run `WarmStartRoutingSpec` from the IDE. I think it does something with paths that I don't fully understand:

```
Caused by: java.lang.NullPointerException
	at scala.collection.mutable.ArrayOps$ofRef$.length$extension(ArrayOps.scala:204)
	at scala.collection.mutable.ArrayOps$ofRef.length(ArrayOps.scala:204)
	at scala.collection.SeqLike.size(SeqLike.scala:108)
	at scala.collection.SeqLike.size$(SeqLike.scala:108)
	at scala.collection.mutable.ArrayOps$ofRef.size(ArrayOps.scala:198)
	at scala.collection.IndexedSeqLike.sizeHintIfCheap(IndexedSeqLike.scala:99)
	at scala.collection.IndexedSeqLike.sizeHintIfCheap$(IndexedSeqLike.scala:99)
	at scala.collection.mutable.ArrayOps$ofRef.sizeHintIfCheap(ArrayOps.scala:198)
	at scala.collection.mutable.Builder.sizeHint(Builder.scala:80)
	at scala.collection.mutable.Builder.sizeHint$(Builder.scala:79)
	at scala.collection.mutable.ArrayBuilder.sizeHint(ArrayBuilder.scala:25)
	at scala.collection.TraversableLike.builder$1(TraversableLike.scala:233)
	at scala.collection.TraversableLike.map(TraversableLike.scala:236)
	at scala.collection.TraversableLike.map$(TraversableLike.scala:230)
	at scala.collection.mutable.ArrayOps$ofRef.map(ArrayOps.scala:198)
	at beam.sim.BeamWarmStart.findFileInDir(BeamWarmStart.scala:169)
	at beam.sim.BeamWarmStart.findRootWarmStartFile(BeamWarmStart.scala:112)
	at beam.sim.BeamWarmStart.rootFile$lzycompute$1(BeamWarmStart.scala:101)
	at beam.sim.BeamWarmStart.rootFile$1(BeamWarmStart.scala:101)
	at beam.sim.BeamWarmStart.getWarmStartFilePath(BeamWarmStart.scala:104)
	at beam.router.RouteHistory.routeHistoryFilePath(RouteHistory.scala:45)
	at beam.router.RouteHistory.loadPreviousRouteHistory(RouteHistory.scala:35)
	at beam.router.RouteHistory.<init>(RouteHistory.scala:27)
```

Also, how can I re-generate the input files for the tests that double or cut in half the travel times? The way R5 imports its network have changed, so the link IDs aren't the same anymore..

**Comment by rajnikantsh** [comment]
For generating double and half the travel times we have to make changes in beam-utilites project. this project consist BeamCalcLinkStats#writeFile method. We have generate these files by making changes in this method

**Comment by michaz** [comment]
Fixed by #1786
